src/plugins/inputs/otr.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function(network, chan, cmd, args) {
+	if (cmd !== "otr") {
+		return;
+	}
+	if (args.length === 0 || args[0] === "") {
```

Comment:
That's not needed, see https://github.com/astorije/shout/blob/master/src/client.js#L225